### PR TITLE
update: readme cleanup and fxes

### PR DESCRIPTION
**Pull request by milosgajdos**
PTAL @thaJeztah @Jamstah

### CODE REVIEW

**Review by Jamstah — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ This repository contains the following components:
...
 | **registry**       | An implementation of the [OCI Distribution Specification](https://github.com/opencontainers/distribution-spec).                                                                                                 |
 | **libraries**      | A rich set of libraries for interacting with distribution components. Please see [godoc](https://pkg.go.dev/github.com/distribution/distribution) for details. **Note**: The interfaces for these libraries are **unstable**. |
-| **documentation**  | Docker's full documentation set is available at [docs.docker.com](https://docs.docker.com). This repository [contains the subset](docs/) related just to the registry.                                                                                                                                          |
+| **documentation**  | Full documentation set is available at [https://distribution.github.io/distribution](https://distribution.github.io/distribution/). This repository [contains the subset](docs/) related just to the registry.                                                                                                                                          |
```
Comment (thaJeztah) [inline README.md:30]: "documentation set" feels a bit awkward.

- Perhaps just "Documentation is available at"
- The second sentence also no longer applies, so I think that whole sentence can be removed.

Comment (milosgajdos) [reply README.md:30]: Yeah I've overlooked that. Will fix

Comment (milosgajdos) [reply README.md]: Fixed, PTAL

**Review by thaJeztah — APPROVED** [review]
LGTM

can you squash?

**Comment by milosgajdos** [comment]
Just squashed, waiting for the CI and then I'll merge.

**Pull request merged**